### ISSUE TITLE
Add tournaments screen

### ISSUE THREAD

**Issue by elrosch** [lichess-org/mobile#535]
Similar to the website and old mobile app a "tournaments" screen would be great.

This features allows to see tournaments that are upcoming, currently in progress or completed.
Users are able to join and participate in tournaments.

**Comment by vaibhavsection** [comment]
Yes, please add **tournaments** screen.

**Comment by tom-anders** [comment]
@veloce I'd be interested in doing some work on this in the future - One question, do you think it would be fine to start with the tournament overview as a list, like in the old app? (see screenshot below.) This would probably be much easier to implement than the timeline view from the web UI (which we can always add later of course)

![image](https://github.com/user-attachments/assets/29ae46f4-b73b-40a3-b656-df42a4fbcbf3)

**Comment by Boubou78000** [comment]
Duplicate of #535

**Comment by veloce** [comment]
It would be interesting to step back a little and think about the UI/UX. How can we improve things from the old app? I agree we shouldn't do the timeline and the list was fine. But new idea are also welcome.

What would be great is to understand how people use this feature from a mobile perspective. For instance we don't show the timeline but we can schedule notifications. What makes more sense from a mobile?

**Comment by tom-anders** [comment]
Yeah good point, maybe we could have a discussion in Discord to get some more ideas from the users?

A notification when you get a new pairing in a running tournament would certainly be very nice. Maybe also a notification when a tournament you registered for is about to start?

**Comment by teefoox** [comment]
The notification when a tournament you registered for is about to start would be great.
The notification for a new pairing is more questionable. I believe that if you leave the tournament page on website it automatically pauses the tournament (it stops searching a new opponent). For the new app, will it continue to search a new opponent even if you leave the tournament page and for example check your Lichess profile ? And what about if you go on another app ?

Apart from that, as a user, I would appreciate the following features : 
- Search a tournament by its name
- Have a page "My tournaments" where I can see the previous, current, and future tournaments I play.
- Have some tournaments suggested by Lichess (maybe a widget on the main page ? The website puts them forward like that and I think it's cool, otherwise I would never play in them.)

**Comment by tom-anders** [comment]
I agree that a tournament widget on the home page would be a nice idea :)

**Comment by aslyamov** [comment]
I think that the tournaments page should definitely have filters by time, chess type, tournament type. It is also possible to display active tournaments in which we participate at the top. Another section could be with tournaments from clubs in which the user is a member.

**Comment by ijm8710** [comment]
Tom had great start to tournaments.

No rush on when v2 comes along as this comment is moreso for my own records of things I'd ultimately like to see

- dive on players recent results
- top players performance/win rate 
- click player to see their profile perf ratings 
- chat
